radix-engine/src/utils/flash_state_generator.rs

```diff
@@ -224,3 +231,37 @@ pub fn generate_seconds_precision_state_updates<S: SubstateDatabase>(db: &S) ->
         ),
     }
 }
+
+pub fn generate_validator_fee_fix_state_updates<S: SubstateDatabase>(db: &S) -> StateUpdates {
```

Comment:
This is a specific change for a specific mainnet protocol update, doesn't seem quite right to keep it in `utils`. Looks like we'll need some notion of the actual protocol updates here